solidstart-tanstackquery-tailwind-modules/src/components/UserProfile/index.tsx

```diff
@@ -9,103 +9,100 @@ import { Icon } from 'solid-heroicons';
 import { TwitterIcon } from '../Icons';
 import OrgList from './OrgList';
 import styles from './UserProfile.module.css';
-import { Show } from 'solid-js';
+import { Show, splitProps } from 'solid-js';
+import { UserProfileProps } from '~/types/user-profile-type';
 
-export interface UserProfileProps {
-  avatarUrl: string;
-  name: string;
-  username: string;
-  bio: string;
-  followers: number;
-  following: number;
-  starredRepos: number;
-  company: string;
-  location: string;
-  websiteUrl: string;
-  twitterUsername: string;
-  organizations: {
-    nodes: {
-      avatarUrl: string;
-      name: string;
-      login: string;
-    }[];
-  };
-}
+const UserProfile = (props: UserProfileProps) => {
+  const [local] = splitProps(props, [
```

Comment:
Is splitting here needed? It looks like you take ever property in the `local`

Reply:
Not really. updating it.

Reply:
Splitting actually isn't needed here.